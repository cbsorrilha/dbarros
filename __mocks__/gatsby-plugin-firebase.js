module.exports = {
    FirebaseContext: {
        _context: {
            Consumer: null,
            Provider: null,
        },
    },
}