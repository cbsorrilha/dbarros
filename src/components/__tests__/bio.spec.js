import React from "react"
import renderer from "react-test-renderer"
import Bio from '../bio';
import { useStaticQuery } from '../../../__mocks__/gatsby';


describe('Snapshot testing the Bio', () => {
    it("renders correctly", () => {
        const props = {}
        useStaticQuery.mockReturnValue({
            site:{
                siteMetadata: {
                    profilePic: 'img',
                    social: {
                        github: {
                            icon: 'img',
                            link: 'a'
                        }
                    }
                }
            }
        });
        const tree = renderer.create(<Bio {...props} />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
