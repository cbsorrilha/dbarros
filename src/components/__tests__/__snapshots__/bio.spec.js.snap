// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing the Bio renders correctly 1`] = `
<div
  className="sc-AykKC gPOpzA"
>
  <div
    className="sc-AykKD jPscvs"
  >
    <h2>
      Olá, eu sou o Cesar
    </h2>
    <p>
      Sou desenvolvedor web desde 2013 e meu sonho é transformar a Baixada Fluminense (RJ) no novo Vale do Silício™.
    </p>
    <p>
      Desenvolvo no back-end com NodeJS e um pouquinho de Python, no front-end com React, Vue e o que mais vier e sei até um pouquinho de infra com docker e kubernetes.
    </p>
    <p>
      Também me meto escrever na internet.
    </p>
    <p>
      Quer saber mais? Fala comigo.
    </p>
    <div
      className="sc-AykKE dnwoWY"
    >
      <div
        className="sc-AykKF GwDrs"
      >
        <a
          href="a"
        >
          <test-file-stub
            style={
              Object {
                "color": "#2cb67d",
              }
            }
          />
        </a>
      </div>
    </div>
  </div>
  <img
    alt="Eu"
    src="img"
    title="Eu sou o da direita"
  />
</div>
`;
